packages/main/src/types/TableCellHorizontalAlign.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Alignment of the &lt;ui5-table-cell&gt; component.
+ *
+ * @public
+ */
+enum TableCellHorizontalAlign {
+    /**
+     * @public
+     */
+    Left = "Left",
+
+    /**
+     * @public
+     */
+    Begin = "Start",
```

Comment:
Should this actually be called Begin and not Start, if all justify-content values are allowed?

Reply:
You're right. I didn't change it because I was awaiting further review from @aborjinik. But I've changed it now :)